safe/definitions/layer_purposes.py

```diff
@@ -161,7 +161,7 @@
         'includes totals for affected and not affected status. It is used '
         'to generate reports and can be exported to a spreadsheet for '
         'further analysis.'),
-    'allowed_geometries': [],
+    'allowed_geometries': None,  # It's a table.
```

Comment:
Is there any pro using `None` instead empty list? Since empty list is consistent with the other allowed_geometries as list.